mirdata/jams_utils.py

```diff
@@ -86,15 +90,14 @@ def jams_converter(
                 + 'for this track cannot be found, and it is required'
                 + 'to compute duration.'
             )
+    if spectrum_cante100_path is not None:
+        if audio_path is None:
+            duration = metadata['duration']
 
     # metadata
     if metadata is not None:
         for key in metadata:
-            if (
-                key == 'duration'
-                and duration is not None
-                and metadata[key] != duration
-            ):
+            if key == 'duration' and duration is not None and metadata[key] != duration and audio_path is not None:
```

Comment:
is this a black formatting issue?

Reply:
I think so. Does it mean a problem? I could write it in a single line.